server.py

```diff
@@ -31,12 +31,26 @@ def set_logging_state():
     logging.debug('Serving at http://%s:%s' % (cfg['host'], cfg['port']))
 
 
+async def build_docs():
+    process = await asyncio.create_subprocess_exec('sphinx-build', 'docs/', 'docs/_build/html',
+                                                   '-b', 'html', '-c', 'docs/',
+                                                   stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE)
+    stdout, stderr = await process.communicate()
+    if process.returncode != 0:
+        print('Unable to refresh docs')
+        if cfg['debug']:
+            print(stderr)
```

Comment:
let's swap this from print to logging.debug().. you can do it against the root logger (just import logging)

Reply:
I switched all the print statements here to go through logging instead.